openssl/src/sign.rs

```diff
@@ -78,6 +79,29 @@ use ffi::{EVP_MD_CTX_free, EVP_MD_CTX_new};
 #[cfg(any(ossl101, ossl102))]
 use ffi::{EVP_MD_CTX_create as EVP_MD_CTX_new, EVP_MD_CTX_destroy as EVP_MD_CTX_free};
 
+/// Salt lengths that must be used with `set_rsa_pss_saltlen`.
+pub enum RsaPssSaltlen {
```

Comment:
We'll probably want to keep this extensible - might as well use the same newtype wrapper approach as we did for Oid - cosnts for -1 and -2 and a constructor for custom.